rfcs/013-introduce-headless-components-library.md

```diff
@@ -0,0 +1,50 @@
+
+## Motivation
+
+- Reduce the load on the engineering team by re-using components already built with UX/accessibility best practices
+- Build new UI more easily, through a library of components that are already tested and documented
+- Prepare the ground for introducing a [CSS library](https://github.com/opencollective/opencollective/pull/6694)
+
+## Solution
+
+<!--
+Describe the proposed solutions with facts and arguments in favor of this change.
+If it has some drawbacks, include them as well.
+-->
+
+## Alternatives
+
+|                                   | Material Base                 | Headless UI | PrimeReact | Radix UI                         | ReaKit | Reach UI | React Aria |
+| --------------------------------- | ----------------------------- | ----------- | ---------- | -------------------------------- | ------ | -------- | ---------- |
+| Popularity (weekly npm downloads) | 3.3m                          | 1.1m        | 87k        | 1.9m (@radix-ui/react-primitive) | 165k   | n/a      | 163k       |
+| Maturity                          | In beta (introduced Sep 2022) |             |            | Mature (first release Dec 2020)  |        |          |            |
+| Maintenance                       |                               |             |            | Active                           |        |          |            |
+| License                           | MIT                           | MIT         | MIT        | MIT                              |        | MIT      | Apache 2.0 |
+| Tailwind Comp.                    | ✅                            | ✅          |            | ✅                               |        |          |            |
+| **Supported components**          |                               |             |            |                                  |        |          |            |
+| Modal                             | ✅                            | ✅          |            | ✅                               |        |          |            |
+| Checkbox                          | ❌                            | ❌          |            | ✅                               |        |          |            |
+| Toggle                            | ✅                            | ✅          |            | ✅                               |        |          |            |
+| Tooltip                           | ❌                            | ❌          |            | ✅                               |        |          |            |
+| Select                            | ✅                            | ✅          |            | ✅                               |        |          |            |
+| Dropdown                          | ✅                            | ✅          |            | ✅                               |        |          |            |
+| Collapsible                       | ❌                            | ❌          |            | ✅                               |        |          |            |
```

Comment:
@gustavlrsn According to https://github.com/material-components/material-components-web/tree/master, Material does have these components. Did you pick the list from a different place?

```suggestion
| Checkbox                          | ✅                            | ❌          |            | ✅                               |        |          |            |
| Toggle                            | ✅                            | ✅          |            | ✅                               |        |          |            |
| Tooltip                           | ✅                            | ❌          |            | ✅                               |        |          |            |
| Select                            | ✅                            | ✅          |            | ✅                               |        |          |            |
| Dropdown                          | ✅                            | ✅          |            | ✅                               |        |          |            |
| Collapsible                       | ❌                            | ❌          |            | ✅                               |        |          |            |
```

Reply:
@Betree that is another library, here is the one I've looked at and it's list of components: https://mui.com/base-ui/getting-started/overview/

Confusing since it's very similar names, will update the title in the table from Material Base to @mui/base (which is the headless version of @mui/material that we have in the product).